src/web/collectors/CollectorRestartButton.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+
+import CollectorsActions from 'collectors/CollectorsActions';
+
+const CollectorsRestartButton = React.createClass({
```

Comment:
The component is called `CollectorsRestartButton` but the file is using the singular `Collector` which makes it difficult to grep for it.